app/Resources/translations/messages.nl.xliff

```diff
@@ -0,0 +1,241 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="note">
+                <source>note</source>
+                <target>NOTITIE</target>
```

Comment:
The usage of note in the templates is more like "AANTEKENING" than "NOTITIE". I also think "NOOT" can be used here as well.